releasenotes/notes/add-alerts-for-swap-availability-75e28ed7f913d1ec.yaml

```diff
@@ -2,7 +2,7 @@
 features:
   - |
     Added two alerts (Warning and critical) that are triggered when the ratio
-    of (free_swap_sppace / total_swap_space) is below thresholds.
+    of (free_swap_space / total_swap_space) is below thresholds.
     Each threshold can be modified by alterting value of
```

Comment:
You should fix the typo of altering while here.

Reply:
Done